test/errhandling/psahabu/catch-parens.chpl

```diff
@@ -0,0 +1,19 @@
+use ThrowError;
+
+try {
+  throwAnError();
+} catch () {
```

Comment:
That said, I don't have a problem with it being legal for now. But I'd probably rule it out in the parser. To do that, you could split catch_expr into catch_expr_nonempty and catch_expr_empty and just have the parenthesis version use catch_expr_nonempty. Up to you.